documentation/book/con-what-the-topic-operator-does.adoc

```diff
@@ -27,7 +25,7 @@ And also, in the other direction:
 * if a topic is deleted from the Kafka cluster, the operator will create the `KafkaTopic` describing it
 * if a topic in the Kafka cluster is changed, the operator will update the `KafkaTopic` describing it
 
-This is beneficial to a {ProductPlatformName}-centric style of deploying applications because it allows you to declare a `KafkaTopic` as part of your application's deployment and the operator will take care of creating the topic for you.
+This allows you to declare a `KafkaTopic` as part of your application's deployment and the operator will take care of creating the topic for you.
```

Comment:
We might want to say 'Topic Operator' here.  We have a Cluster Operator and a Topic Operator and so when we talk about 'the operator' it really could be either of those.  I'm willing to be disagreed with :)